charts/sourcegraph/templates/indexed-search/indexed-search.StatefulSet.yaml

```diff
@@ -55,14 +55,8 @@ spec:
         - name: {{ $name }}
           {{- $item | toYaml | nindent 10 }}
         {{- end }}
+        {{- include "sourcegraph.otelCollectorEnv" . | nindent 8 }}
         {{- if .Values.otelCollector.enabled }}
-        # OTEL_AGENT_HOST must be defined before OTEL_EXPORTER_OTLP_ENDPOINT to substitute the node IP on which the DaemonSet pod instance runs in the latter variable
-        - name: OTEL_AGENT_HOST
-          valueFrom:
-            fieldRef:
-              fieldPath: status.hostIP
-        - name: OTEL_EXPORTER_OTLP_ENDPOINT
-          value: http://$(OTEL_AGENT_HOST):4317
         - name: OPENTELEMETRY_DISABLED
           value: "false"
```

Comment:
I guess zoekt has otel disabled by default?

Reply:
disabled = false => enabled :)

Reply:
lol I just noticed it's `DISABLED` instead of `ENABLED`